agent/internal/client/redacting_sender_test.go

```diff
@@ -0,0 +1,58 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRedactingSender(t *testing.T) {
+	for name, test := range map[string]struct {
+		substitutions map[string]string
+		inputString   string
+		expected      string
+	}{
+		"MultipleSubstitutions": {
+			substitutions: map[string]string{
+				"secret_key": "secret_val",
+			},
+			inputString: "secret_val secret_val",
+			expected:    "<redacted:secret_key> <redacted:secret_key>",
```

Comment:
These tests need to be updated, maybe use `fmt.Sprintf(redactedVariableTemplate, <val>)` instead of hardcoding it?

Reply:
done